environment.py

```diff
@@ -105,13 +105,9 @@ def remote_cmd(cmd, host=None, sudo=None, **kwargs):
 
         inventory = None
 
-        if hasattr(context, 'dynamic_hosts'):
-            host = context.dynamic_hosts
+        if context.inventory == "dyanmic":
```

Comment:
Shoot!  Well, this means all my testing was for naught.

Will fix and re-test ASAP.

----- Original Message -----
From: "Aaron Weitekamp" <notifications@github.com>
To: "aweiteka/UATFramework" <UATFramework@noreply.github.com>
Cc: "Micah Abbott" <miabbott@redhat.com>
Sent: Friday, January 23, 2015 11:15:51 AM
Subject: Re: [UATFramework] Inventory#15 (#17)

> @@ -105,13 +105,9 @@ def remote_cmd(cmd, host=None, sudo=None, **kwargs):
>  
>          inventory = None
>  
> -        if hasattr(context, 'dynamic_hosts'):
> -            host = context.dynamic_hosts
> +        if context.inventory == "dyanmic":

@miabbott "dyanmic" typo. :)

---
Reply to this email directly or view it on GitHub:
https://github.com/aweiteka/UATFramework/pull/17/files#r23459808